apps/web/.env.local.example

```
@@ -31,4 +31,13 @@ LIVE_STATE_WS_URL=
-VITE_REFLAG_PUB_KEY=
\ No newline at end of file
+VITE_REFLAG_PUB_KEY=
+
+# GitHub
+VITE_GITHUB_CLIENT_ID=
+
+# Your ngrok URL pointing to the API server (port 3334)
+VITE_API_URL=https://demetrice-nongreasy-aubrianna.ngrok-free.dev
+
+# Frontend base URL (optional, defaults to http://localhost:3000)
+VITE_BASE_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)